python/datasources/cdc_restricted_local.py

```diff
@@ -117,6 +117,26 @@
 DEATH_DATA_SUPPRESSION_STATES = ("HI", "NE", "SD", "DE")
 
 
+def combine_race_eth(df):
+    """Combines the race and ethnicity fields into the legacy race/ethnicity category.
+       We will keep ths in place until we can figure out a plan on how to display
+       the race and ethnicty to our users disagregted."""
+
+    def get_combined_value(row):
+        if row[ETH_COL] == 'Hispanic/Latino':
+            return std_col.Race.HISP.value
+
+        elif row[RACE_COL] in {'NA', 'Missing', 'Unknown'} or row[ETH_COL] in {'NA', 'Missing', 'Unknown'}:
+            return std_col.Race.UNKNOWN.value
+
```

Comment:
shouldn't `race_col == NA` AND `eth_col == NA` result in "missing" which would be `NA` ?

Reply:
That all equates to `UNKNOWN` to us